src/components/project/project.service.ts

```diff
@@ -624,7 +624,7 @@ export class ProjectService {
           node('prop', 'Property', { active: true }),
         ])
         .with('*')
-        .orderBy(`prop.value ${input.order}`);
+        .orderBy(`LOWER(prop.value) ${input.order}`);
```

Comment:
FYI this is fine for sorting by name, but it won't work for sorting by sensitivity or dates

Reply:
@CarsonF correct. I just tested with sensitivity, which cypher interpreted as a string (I suppose) so nothing bad happened there, but sorting by dates broke it.